dusty/commands/status.py

```diff
@@ -0,0 +1,19 @@
+from prettytable import PrettyTable
+
+from ..compiler.spec_assembler import get_assembled_specs
+from ..log import log_to_client
+from ..systems.docker import get_dusty_containers
+
+def _has_activate_container(spec_type, service_name):
+    if spec_type == 'libs':
+        return False
+    return get_dusty_containers([service_name]) != []
+
+def get_dusty_status():
+    assembled_specs = get_assembled_specs()
+    table = PrettyTable(["Name", "Type", "Has Active Container"])
+    for spec_type in ['app', 'lib', 'service']:
```

Comment:
Instead of doing this, I'd advocate writing a method that goes on `assembled_specs` class, which returns a list or a set of the specs you need.  Those will have `.name` and `.spec_type` attribute, which you will put in your table